### ISSUE TITLE
kubeapps-dashboard-ui crash loop on CentOS deployed kubernetes 1.8.5 

### ISSUE THREAD

**Issue by kgignatyev-inspur** [kubeapps/kubeapps#100]
kubernetes deployed on AWS with kops, and CentOS image was specified for machines per https://github.com/kubernetes/kops/blob/master/docs/images.md

 kubectl -n kubeapps logs kubeapps-dashboard-ui-5dc47b8d6d-s4qbk

Welcome to the Bitnami nginx container
Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-nginx
Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-nginx/issues
Send us your feedback at containers@bitnami.com

nami    INFO  Initializing nginx
Error executing 'postInstallation': EEXIST: file already exists, symlink '/bitnami/nginx/conf' -> '/opt/bitnami/nginx/conf'

 kubectl version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.0", GitCommit:"925c127ec6b946659ad0fd596fa959be43f0cc05", GitTreeState:"clean", BuildDate:"2017-12-16T03:16:50Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.5", GitCommit:"cce11c6a185279d037023e02ac5249e14daa22bf", GitTreeState:"clean", BuildDate:"2017-12-07T16:05:18Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}

**Comment by prydonius** [comment]
cc @tompizmor

**Comment by ngtuna** [comment]
cc/ @sameersbn can you look at this ?

**Comment by myspotontheweb** [comment]
I was getting this error when running minikube with version 1.9.4 of Kubernetes.  Upgrading to 1.9.6 appears to solve the problem. No explanation just weird.

```
$ minikube version
minikube version: v0.28.0

$ minikube config view
- vm-driver: kvm2
- kubernetes-version: v1.9.6
- memory: 4096
```

**Comment by kgignatyev-inspur** [comment]
v0.27.0 working for me now and I cannot experiment at the moment because in China pulling artifacts like dashboard from google is very difficult

**Comment by jiaj12** [comment]
k8s 1.11 installed by kubeadm on CentOS 7 , same problem.
Error executing 'postInstallation': EEXIST: file already exists, symlink '/bitnami/nginx/conf' -> '/opt/bitnami/nginx/conf'

I think the problem could result in the problem by bitnami/nginx image doesn't support CentOS 7 with overlayFS enabled.

I guess the solution is either change the overlayFS or change the nginx image.

Still testing

**Comment by prydonius** [comment]
@jiaj12 ah that might very well be it, I have seen this issue before. You could try switching to aufs. cc @sameersbn

**Comment by sameersbn** [comment]
@jiaj12 It's been a while since I have tried this setup, but from what I remember RHEL treats OverlayFS as a preview filesystem. The following document https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/7.2_Release_Notes/technology-preview-file_systems.html lists the problems and possible workarounds for using OverlayFS with Docker on RHEL 7.2.

I am not sure if the newer version have fixed this. I found this article re: your use case, but unfortunately it's only accessible to paying RHEL users : https://access.redhat.com/solutions/3156851
